stable/aws-for-fluent-bit/README.md

```diff
@@ -38,7 +38,7 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `serviceAccount.create` | Whether a new service account should be created | `true` |
 | `service.extraService` | Append to existing service with this value | `""` |
 | `service.parsersFiles` | List of available parser files | `/fluent-bit/parsers/parsers.conf` |
-| `service.additionalParsers` | Adding more parsers with this value | `""` |
+| `service.extraParsers` | Adding more parsers with this value | `""` |
```

Comment:
renaming the key seems like not a good idea, won't this break folks who depend on it??

Reply:
Sorry I didn't read the description- you're just updating the docs to reflect what the actual template supports right?